### ISSUE TITLE
A question about the tokenizer

### ISSUE THREAD

**Issue by 02bbt** [ATR-DBI/ScanQA#4]
Hello, I am puzzled with the performance of the bert-based-uncased tokenizer. It seems to be a lot worse than glove's result. I wonder whether I have mistaken it or not.

**Comment by miyatai2** [comment]
How about using the option `--finetune_bert_last_layer' or using the lower learning rate 1e-4?
